src/main/java/redis/clients/jedis/JedisClusterCommand.java

```diff
@@ -85,7 +100,10 @@ public T runWithAnyNode() {
   }
 
   private T runWithRetries(final int slot) {
+    Instant deadline = Instant.now().plus(maxTotalRetriesDuration);
```

Comment:
Taking the the time on each successful call seems like a waste and might impact the performance.

Reply:
@gkorland According to https://www.alibabacloud.com/blog/performance-issues-related-to-localdatetime-and-instant-during-serialization-operations_595605 

Throughput of Instant.now+atZone+**format**+DateTimeFormatter.ofPattern is 6816922.578 ops/sec.
Without any formatting, throughput of Instant.now+plus should be much higher. Shouldn't it be enough?